src/components/Modal/src/ModalArrow.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+	<span
```

Comment:
is this extra `span` necessary? why not just put these styles on the inner component?